### PR TITLE
Prevent card from going to decision release when its paused

### PR DESCRIPTION
**Pull request by dhaselhan**
* Simple front end rule for now until we get more rule requirements
* Fix error toast to have dismiss button by default, change button font color to grey

### CODE REVIEW

File: alcs-frontend/src/app/shared/drag-drop-board/drag-drop-board.component.ts [hunk 1]
```diff
@@ -32,6 +35,14 @@ export class DragDropBoardComponent {
       return;
     }
 
+    debugger;
```
Comment (mhuseinov) [inline alcs-frontend/src/app/shared/drag-drop-board/drag-drop-board.component.ts:38]: we probably do no need this

File: alcs-frontend/src/app/shared/drag-drop-board/drag-drop-board.component.ts [hunk 2]
```diff
@@ -32,6 +35,14 @@ export class DragDropBoardComponent {
...
+    if (targetColumn.status === 'RELE' && selectedCard.paused) {
+      this.toastService.showErrorToast(
+        "Paused cards cannot be moved to the 'Decision Released' column. Please unpause the card first and try again"
```
Comment (mhuseinov) [inline alcs-frontend/src/app/shared/drag-drop-board/drag-drop-board.component.ts:41]: is there any enum that contains description "Decision Released" ?

Comment (dhaselhan) [reply alcs-frontend/src/app/shared/drag-drop-board/drag-drop-board.component.ts:41]: I could add a subscriber to the applicationStatus from the service, look up the value from there using the code, and use that label?

File: alcs-frontend/src/app/shared/drag-drop-board/drag-drop-board.component.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/shared/drag-drop-board/drag-drop-board.component.ts]: OOF, removed.

**Review by mhuseinov — APPROVED** [review]
🎯

**Pull request merged**